scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -250,12 +257,16 @@ trait DocumentOps { self: DatabaseOps =>
               // Unfortunately, this is often not the case as demonstrated by a bunch of cases above and below.
               if (tryMpos(gstart, gend)) return
 
+              pprint.log(gtree)
```

Comment:
:D

Reply:
Removed.

Reply:
I should install a pre-push hook for this 😄